gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -79,22 +82,48 @@ public class HttpJsonOperationsStub extends OperationsStub {
   private static final String LRO_CANCEL_OPERATION =
       "google.longrunning.Operations.CancelOperation";
 
+  private final FieldsExtractor<ListOperationsRequest, Map<String, String>>
+      LRO_LIST_PATH_EXTRACTOR =
+          request -> {
+            Map<String, String> fields = new HashMap<>();
+            ProtoRestSerializer<ListOperationsRequest> serializer = ProtoRestSerializer.create();
+            serializer.putPathParam(fields, "name", request.getName());
+            return fields;
+          };
+  private final FieldsExtractor<GetOperationRequest, Map<String, String>> LRO_GET_PATH_EXTRACTOR =
+      request -> {
+        Map<String, String> fields = new HashMap<>();
+        ProtoRestSerializer<GetOperationRequest> serializer = ProtoRestSerializer.create();
+        serializer.putPathParam(fields, "name", request.getName());
+        return fields;
+      };
+
+  private final FieldsExtractor<DeleteOperationRequest, Map<String, String>>
+      LRO_DELETE_PATH_EXTRACTOR =
+          request -> {
+            Map<String, String> fields = new HashMap<>();
+            ProtoRestSerializer<DeleteOperationRequest> serializer = ProtoRestSerializer.create();
+            serializer.putPathParam(fields, "name", request.getName());
+            return fields;
+          };
+
+  private final FieldsExtractor<CancelOperationRequest, Map<String, String>>
+      LRO_CANCEL_PATH_EXTRACTOR =
+          request -> {
+            Map<String, String> fields = new HashMap<>();
+            ProtoRestSerializer<CancelOperationRequest> serializer = ProtoRestSerializer.create();
+            serializer.putPathParam(fields, "name", request.getName());
+            return fields;
+          };
```

Comment:
OperationsStub has a default FieldsExtractor for each RPC (difference being that the request type for each FieldsExtractor is different). Each request type in the proto file has the `name` field, so we should be able to the default FieldsExtractor even for custom paths: https://github.com/googleapis/googleapis/blob/d5a51f6baba66f5dde236211d41d66efa1e4e7ca/google/longrunning/operations.proto#L171